src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Manipulation_Tests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Input
+{
+	public class Manipulation_Tests : SampleControlUITestBase
+	{
+		[Test]
```

Comment:
Non uniform indentation.

Reply:
It seems to be a bug of the github compare tool:
![image](https://user-images.githubusercontent.com/8635919/68426236-accae180-0175-11ea-8cb0-736bf8256472.png)